test/mosaic-integration-tests/pom.xml

```diff
@@ -53,6 +53,19 @@
             <version>${mosaic.version}</version>
             <scope>test</scope>
         </dependency>
+        <!--For applications-->
```

Comment:
@kschrab dependencies fine like this?

Reply:
Apparently not since build fails 😅

Reply:
At the moment I just put the applications for the new integration test into the package `org.eclipse.mosaic.test.sendandreceive`, which requires the new dependencies. Should the apps rather be put in a separate module, which is then added as a dependency?

Reply:
Alright. Seems to be fixed now.

Reply:
`org.eclipse.mosaic.test.app.sendandreceive`

und vllt den test lieber "SendAndReceiveIT" oder vllt "CellSnsSendAndReceiveIT" nennen."CommunicationSimulatorsIT" finde ich etwas zu allgemein, OMNeT++/ns-3 sind ja nicht dabei.